charts/stable/traefik-forward-auth/ci/ct-values.yaml

```
@@ -1,5 +1,3 @@
-providers:
-  google:
-    enabled: true
-    clientId: "fakeclientid"
-    clientSecret: "fakeclientsecret"
+env:
+  PROVIDERS_GOOGLE_CLIENT_ID: fakeclientid
+  PROVIDERS_GOOGLE_CLIENT_SECRET: fakeclientsecret
```

```suggestion
  PROVIDERS_GOOGLE_CLIENT_SECRET: fakeclientsecret

middleware:
  enabled: false
```

It makes sense to enable the middleware by default, but in our CI environment we don't have that CRD.